bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -168,5 +196,9 @@
 		<item-type>String</item-type>
 		<label>Scene</label>
 		<description>The scene channel allows recalling a scene to all lights that belong to the scene.</description>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
```

Comment:
From a machine-readability point, I also think those tags don't have much meaning, since a machine can't make any sense from a String item state in terms of what this means for a light.

Reply:
Tags removed